jpa/src/test/java/org/jdbi/v3/jpa/Something.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.jpa;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+
+@Entity
+public class Something extends Somesuper {
+    @Column
+    private String name;
+
+    @Column
+    public Long value;
+
+    @SuppressWarnings("unused")
+    private Something() {
+    }
+
+    public Something(int id, String name, Long value) {
+        super(id);
+        this.name = name;
+        this.value = value;
+    }
+
+    public String name() {
```

Comment:
would be nice to test a method setter / getter pair and a subclass overriding the superclass